package.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "@comfortdelgro/compass-design-monorepo",
   "private": true,
```

Comment:
If this is to close #5, you'll need to add `description`, `repository`, `tags` and other fields to make it complete. See this for reference https://docs.npmjs.com/cli/v9/configuring-npm/package-json.

Reply:
done, hot the package.json is sufficient now